test/modern_bpf/test_suites/syscall_exit_suite/creat_x.cpp

```diff
@@ -0,0 +1,116 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_creat
+
+#if defined(__NR_fstat) && defined(__NR_unlinkat) && defined(__NR_close)
```

Comment:
This is another nitpick that has been itching on me for some time now, we obviously don't need to address it in this PR, but gtest has the [`GTEST_SKIP` macro](http://google.github.io/googletest/advanced.html#skipping-test-execution). Maybe we can use that instead of completely removing tests when a syscall is not defined? That way we can have a complete and clear report on what is and what isn't being run. The macro was added on 1.10, so the embedded gtest should have it defined.

Reply:
Oh, that's a smart idea, we can surely apply this, I don't see any drawback! Again since this involves different PRs probably i would do it at the end of this porting phase, WDYT?

Reply:
Yup, no need to go around changing a bunch of code around, lets do it in a follow up 🙂